app/src/actions/workspace.js

```diff
@@ -215,7 +215,7 @@ function processLegacyWorkspace(data, dispatch) {
     endOuterDate = new Date(serializedStartDate[1]);
     startOuterDate = new Date(endOuterDate.getTime() - workspace.state.timeExtent);
     startInnerDate = startOuterDate;
-    endInnerDate = new Date(startInnerDate.getTime() + Math.min(workspace.state.timeExtent, 15778476000));
+    endInnerDate = new Date(startInnerDate.getTime() + Math.min(workspace.state.timeExtent / 2, 15778476000));
```

Comment:
??

I dont understand what this fixes. Seems to only inject randomness. could you explain?